client/src/utils/utils.js

```diff
@@ -1,3 +1,9 @@
+const isoDateConverter = (isoString) => {
+  const options = { year: "2-digit", month: "2-digit", day: "2-digit" };
+  const date = new Date(isoString);
+  return date.toLocaleDateString("de-DE", options);
+};
+
```

Comment:
There is no need to redefine `options` on every call, move it out of the function.
```suggestion
const options = { year: "2-digit", month: "2-digit", day: "2-digit" };

const isoDateConverter = (isoString) => {
  const date = new Date(isoString);
  return date.toLocaleDateString("de-DE", options);
};
```

I feel the naming could be improved here. `isoDateConverter` sounds like it is a thing (an object) but it is a function that __does__ something. Maybe a verb oriented name like `convertIsoDateToLocale` would be a better fitß